db/migrate/20160726061957_turn_default_value_to_lookup_value.rb

```diff
@@ -0,0 +1,19 @@
+class TurnDefaultValueToLookupValue < ActiveRecord::Migration
+  def up
+    LookupKey.unscoped.each do |key|
```

Comment:
It creates multiple records one by one, can you please wrap it up in a transaction to speed things a bit?
Something like this:
``` ruby
LookupValue.transaction do
  LookupKey.unscoped.each do |key|
    LookupValue.create(:match => 'default', :value => key.default_value, :lookup_key_id => key.id, :use_puppet_default => key.use_puppet_default)
  end
end